components/Modal.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
```

Comment:
h/t to @LayaTaal for this generic modal component that he originally created for https://github.com/texas-justice-initiative/website-nextjs/pull/357. 🙌